lib/middleman-prismic/preview.rb

```diff
@@ -11,11 +11,20 @@ def call(env)
         if req.path =~ %r(^/preview)
           token = req.params["token"]
 
-          succeeded = Kernel.system 'middleman', 'prismic', '--ref', token
-          if succeeded
-            [302, {'Location' => preview_url(token)}, ['Found']]
-          else
-            [500, {'Location' => '/?error=preview_failure'}, ['Error']]
+          begin
+            stdout, stderr, status = Open3.capture3('middleman', 'prismic', '--ref', token)
+
+            if status.success?
+              [302, {'Location' => preview_url(token)}, ['Found']]
+            else
+              [500, {'Location' => '/?error=preview_failure'}, [stdout, stderr]]
```

Comment:
With `stdout` and `stderr` captured separately, the order of the output can get lost. Is that in this context?

Reply:
resolved in 816856c